LL/h_celicain.ttl

```diff
@@ -41,13 +43,14 @@
     a foaf:Person;
     foaf:name "Trenfhir";
     foaf:name "Trenfer";
-    rel:childOf <#Celicain>.
+    rel:childOf <#Celicain>;
+    owl:sameAs <http://example.com/LL/h_mbresail.ttl#Trenfhir>.
 
 <#Celicain>
     a foaf:Person;
     foaf:name "Celicain";
     foaf:name "Ceilecan";
-    rel:childOf <#Garbith>.
+    rel:childOf <#Garbith>;
```

Comment:
It looks like you meant to do something here but stopped.  Did you forget to put the sameAs in?